modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1711,12 +1692,31 @@ Either use xref::indexes-for-search-performance.adoc[] instead, or remove the of
 .Error message
 [source, error]
 ----
-Unable to create Constraint( name='intersectionConstraint', type='RELATIONSHIP KEY', schema=()-[:INTERSECTION {coordinates}]-() ):
-Both Relationship(0) and Relationship(1) have the type `INTERSECTION` and property `coordinates` = {geometry: {type: "Point", coordinates: [1.0, 2.0], crs: {type: link, properties: {href: "http://spatialreference.org/ref/sr-org/7203/", code: 7203}}}}
+Unable to create Constraint( name='knows_level', type='RELATIONSHIP KEY', schema=()-[:KNOWS {level}]-() ):
+Both Relationship(0) and Relationship(1) have the type `KNOWS` and property `level` = 10
 ----
 
 ======
-////
+
+The constraint creation fails on the first offending relationships that are found.
```

Comment:
```suggestion
The constraint creation fails on the first offending relationship that is found.
```

Reply:
Alternative: "The constraint creation fails on the first offending relationship that is found"